distributed/tests/test_hdfs.py

```diff
@@ -263,3 +263,53 @@ def test_read_csv_lazy(s, a, b):
 
         result = yield e.compute(df.id.sum(), sync=False)[0]._result()
         assert result == 1 + 2 + 3 + 4
+
+
+@gen_cluster([(ip, 1), (ip, 1)], timeout=60)
+def test__read_text(s, a, b):
+    with make_hdfs() as hdfs:
+        e = Executor((s.ip, s.port), start=False)
+        yield e._start()
+
+        with hdfs.open('/tmp/test/text.1.txt', 'w') as f:
+            f.write('Alice 100\nBob 200\nCharlie 300'.encode())
+
+        with hdfs.open('/tmp/test/text.2.txt', 'w') as f:
+            f.write('Dan 400\nEdith 500\nFrank 600'.encode())
+
+        with hdfs.open('/tmp/test/other.txt', 'w') as f:
+            f.write('a b\nc d'.encode())
+
+        b = yield _read_text('/tmp/test/text.*.txt',
+                             collection=True, lazy=True)
+        yield gen.sleep(0.5)
+        assert not s.dask
```

Comment:
Yes.  `s.dask` is a record of all computations known to the scheduler.  Here we assert that the scheduler knows nothing.